packages/ui/src/proposals/hooks/useProposal.ts

```diff
@@ -11,6 +11,8 @@ export const useProposal = (id: string): UseProposal => {
 
   const { loading, data } = useGetProposalQuery(params)
 
+  console.log(data)
+
```

Comment:
This query data doesn't contain all proposal data. In case of 3 constitutionals, this only displays 2 constitutionals.
This is evidently an error that is created during proposal creation.